examples/opencode-lmstudio/test.sh

```
@@ -75,28 +77,16 @@ fi
 echo ""
 
 # Test 4: Check endpoint status
-echo -e "${BLUE}[4/8]${NC} Checking LM Studio endpoint status in Olla..."
+echo -e "${BLUE}[4/6]${NC} Checking LM Studio endpoint status in Olla..."
 if ENDPOINTS=$(curl -s -f "http://${OLLA_HOST}:${OLLA_PORT}/internal/status/endpoints" 2>&1); then
```

Shellcheck (0.11.0)

[warning] ENDPOINTS appears unused. Verify use (or export if used externally).

(SC2034)